airflow/dags/gtfs_ckan_uploader/calitp_data_ca_exporter.py

```diff
@@ -0,0 +1,66 @@
+import os
+import requests
+from calitp import get_table
+
+
+#
+API_ENDPOINT = "https://data.ca.gov/api/3/action/resource_update"
+API_KEY = os.environ.get("someAPIKey")
+#
+
+agency = get_table("gtfs_schedule.agency", as_df=True)
+agency.to_csv("gtfs_schedule_agency.csv")
+
+r = requests.post(
+    API_ENDPOINT,
+    data={"id": "e8f9d49e-2bb6-400b-b01f-28bc2e0e7df2"},
+    headers={"Authorization": API_KEY},
+    files={"upload": "gtfs_schedule_agency.csv"},
+)
+
+#
+routes = get_table("gtfs_schedule.routes", as_df=True)
+routes.to_csv("gtfs_schedule_routes.csv")
```

Comment:
let's put these files in `/tmp`

Reply:
Elaborating a bit--can you use the tempfile.TemporaryDirectory? In this case, it looks something like...

```python
with tempfile.TemporaryDirectory() as tmp_dir:
    routes = get_table("gtfs_schedule.routes", as_df=True)
    routes.to_csv(f"{tmp_dir}/gtfs_schedule_routes.csv")
    
    # be sure to include the original post request here!
    r = request.post(...)
```

This will make sure everything is cleaned up super fast / runs in a unique folder!